app-web/src/components/CollectionPreview/CollectionPreview.js

```diff
@@ -20,6 +20,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from '@emotion/styled';
 import { css } from '@emotion/core';
+import { Link } from 'gatsby';
```

Comment:
Prefer importing our homemade Link component which wraps this component
`import { Link } from '../../UI/Link';`

Reply:
For sure, will update that now

Reply:
updated it on the new push, then styled it a bit for our needs - let me know what you think